Functions/BIDS/xASL_bids_GenerateParticipantsJSON.m

```diff
@@ -0,0 +1,152 @@
+listKeys = tableParticipants(1,:);
+
+for iList=1:length(listKeys)
+    switch lower(listKeys{iList})
+
+        
+        %% 3. Parse standard keys
+        case 'participant_id'
+            jsonParticipants.participant_id.Description = 'Unique participant identifier, including _session suffix (==visit in ExploreASL legacy)';
+        case 'session'
+            jsonParticipants.session.Description = 'Unique run identifier (== session in ExploreASL legacy)';
+        case 'age'
+            jsonParticipants.age.Description = 'Age of the participant at the time of scanning';
+            jsonParticipants.age.Units = 'years';
+        case 'sex'
+            jsonParticipants.sex.Description = 'Biological sex of the participant';
+            jsonParticipants.sex.Levels.F = 'female';
+            jsonParticipants.sex.Levels.M = 'male';
+        case 'site'
+            jsonParticipants.site.Description = 'Center where this scan was made';
+            uniqueSites = unique(tableParticipants(2:end, iList));
+            for iSite=1:length(uniqueSites)
+                jsonParticipants.site.Levels.(['Site_' num2str(iSite)]) = uniqueSites{iSite};
+            end
+        case 'gm_vol'
+            jsonParticipants.gm_vol.Description = 'Gray matter volume of the participant at this visit';
+            jsonParticipants.gm_vol.Units = 'liter';
+        case 'wm_vol'
+            jsonParticipants.wm_vol.Description = 'White matter volume of the participant at this visit';
+            jsonParticipants.wm_vol.Units = 'liter';                    
+        case 'csf_vol'
+            jsonParticipants.csf_vol.Description = 'Cerebrospinal fluid volume of the participant at this visit';
+            jsonParticipants.csf_vol.Units = 'liter';
+        case {'gm_icv_ratio', 'gm_icvratio'}
+            jsonParticipants.gm_icv_ratio.Description = 'Ratio of gray matter volume to the intracranial volume of the participant at this visit, proxy of gray matter atrophy';
+            jsonParticipants.gm_icv_ratio.Units = 'ratio';
+        case {'gmwm_icv_ratio', 'gmwm_icvratio'}
+            jsonParticipants.gmwm_icv_ratio.Description = 'Ratio of parenchymal volume (gray matter + WM volume) to the intracranial volume of the participant at this visit, proxy of wholebrain atrophy';
+            jsonParticipants.gmwm_icv_ratio.Units = 'ratio';
+        case 'wmh_vol'
+            jsonParticipants.wmh_vol.Description = 'White matter hyperintensity volume of the participant at this visit';
+            jsonParticipants.wmh_vol.Units = 'liter';
+        case 'wmh_count'
+            jsonParticipants.wmh_count.Description = 'Number of non-connected white matter hyperintensities of the participant at this visit';
+            jsonParticipants.wmh_count.Units = 'integer';
+        case 'meanmotion'
+            jsonParticipants.motion.Description = 'Mean net displacement vector difference (head motion) of this ASL scan';
+            jsonParticipants.wmh_count.Units = 'mm RMS';
```

Comment:
motion not wmh_count

Reply:
Done in a882a8bbda1b10a30e0d3bbadb30ba880788f4ad